api.js

```diff
@@ -165,10 +167,14 @@ Api.prototype._prefixTitle = function (file) {
 	return prefix;
 };
 
-Api.prototype.run = function (files) {
+Api.prototype.run = function (files, forceHasExclusive) {
```

Comment:
I think the second argument would be better as an optional options object with an option called `forceHasExclusive`. 

http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html